components/ui/CompactFeature.vue

```diff
@@ -1,51 +1,63 @@
 <template>
-  <section class="compact-feature">
+  <article class="compact-feature">
     <img
-      v-if="icon"
       class="compact-feature__icon"
-      :src="`/images/icons/icon${icon}White@3x.png`"
+      :src="`/images/icons/${icon}`"
+      alt=""
     >
-    <h2 v-if="title" class="compact-feature__title">
+    <h3 class="compact-feature__title">
       {{ title }}
-    </h2>
-    <p v-if="description" class="compact-feature__description">
+    </h3>
+    <p class="compact-feature__description">
       {{ description }}
     </p>
-  </section>
+  </article>
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
 import { Component, Prop } from 'vue-property-decorator'
 
 @Component
-export default class CompactFeature extends Vue {
-  @Prop(String) icon: string | undefined
-  @Prop(String) title: string | undefined
-  @Prop(String) description: string | undefined
+export default class extends Vue {
+  @Prop(String) icon!: string
```

Comment:
Be careful, if the property is not required, it **can be `undefined`** and the type should reflect that.